features/step_definitions/admin_steps.rb

```diff
@@ -58,7 +58,12 @@ def update_settings(settings)
 
 Given /^basic languages$/ do
   Language.default
-  Language.find_or_create_by_short_and_name("DE", "Deutsch")
+  german=Language.find_or_create_by_short_and_name("DE", "Deutsch")
+  de=Locale.new
+  de.iso='de'
+  de.name='Deutsch'
```

Comment:
Surrounding space missing for operator '='.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.